src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ CHANGELOG
 -----
 
  * Removed support to load translation resources from the legacy directories `src/Resources/translations/` and `src/Resources/<BundleName>/translations/`
+ * Removed `Symfony\Bundle\FrameworkBundle\Controller\ControllerNameParser`.
+ * Removed `ResolveControllerNameSubscriber`
 
```

Comment:
please add 
> * Removed support for `bundle:controller:action` and `service:action` syntaxes to reference controllers. Use `serviceOrFqcn::method`

to match the UPGRADE entries added in https://github.com/symfony/symfony/pull/26085

Reply:
@chalasr deprecation of `service:action`  has been reverted

Reply:
I wasn't sure, thanks